src/snowflake/cli/plugins/object/commands.py

```diff
@@ -92,3 +104,45 @@ def describe(
     return QueryResult(
         ObjectManager().describe(object_type=object_type, name=object_name)
     )
+
+
+def _parse_object_definition(object_definition: List[str]) -> Dict[str, Any]:
+    import json
+
+    def _parse_list_to_dict(object_definition: List[str]) -> Dict[str, Any]:
+        payload = {}
+        for item in object_definition:
+            try:
+                key, value = item.split("=", 1)
```

Comment:
Should we be able to use `parse_key_value_variables` as we do in `snow sql` and `snow stage execute`?

Reply:
Nice catch, didn't know about that one!

Reply:
fixed